src/main/java/echo/Client.java

```diff
@@ -0,0 +1,42 @@
+package echo;
+
+import java.io.*;
+import java.net.*;
+import java.nio.Buffer;
+
+import static java.lang.System.*;
+
+public class Client {
+
+    public static void main(String[] args) throws IOException {
+
+        runClient(args[0],Integer.parseInt(args[1]));
+    }
+
+    public static void runClient (String hostname, int portNumber){
+        try (
+                Socket clientSocket = new Socket(hostname, portNumber);
+                PrintWriter output = new PrintWriter(clientSocket.getOutputStream(), true);
+                BufferedReader input = new BufferedReader(new InputStreamReader(clientSocket.getInputStream()));
+                BufferedReader stdInput = new BufferedReader(new InputStreamReader(System.in));
```

Comment:
are they both needed?

Reply:
Not too sure but I've removed the client now.